readme.md

```diff
@@ -4,6 +4,10 @@ Providing you with all sorts of Good Eggs-style fake data!
 
 All exported functions are stateless (meaning you don't have to worry about binding to `this`).
 
+## Purpose and Scope
```

Comment:
Maybe it's implied in this paragraph as is, but I wonder if we could be more explicit about the intended action to take when your use case can't be adequately covered by fake-eggs? Something like:
> Use fake-eggs to generate all [...]
> 
> If you encounter a substantial use case where fake-eggs is unable to expressively generate random factory data, please consider opening a PR enhancing it or opening an Issue to track it.

Reply:
Good idea - I support a call-to-action of sorts!